tests/unit/datarobot_drum/drum/language_predictors/test_base_language_predictor.py

```diff
@@ -116,7 +116,7 @@ def mock_dr_client(self):
             yield
 
     def test_mlops_init(self, language_predictor_with_mlops, mock_mlops):
-        mock_mlops.set_channel_config.called_once_with("spooler_type=API")
+        # mock_mlops.set_channel_config.assert_called_once_with("spooler_type=API")
```

Comment:
Why is this being commented out? It looks like the old code wasn't actually doing what the original author thought it was doing.

Reply:
This assert was not working: `called_once_with` is not an assert method. When I renamed it, it started failing. I shared this https://datarobot.slack.com/archives/C01BAAV38FR/p1738236525603729?thread_ts=1738164380.474679&cid=C01BAAV38FR in the original thread where we discussed this upgrade, but had no response

Reply:
I'd created a bug ticket and add that to the commment.